presto-docs/src/main/sphinx/security/server.rst

```diff
@@ -0,0 +1,254 @@
+==========================================
+Presto Coordinator Kerberos Authentication
+==========================================
+
+The Presto server can be configured to enable Kerberos authentication over
+HTTPS for clients such as the :doc:`Presto CLI </security/cli>` or the various
+JDBC and ODBC drivers.
+
```

Comment:
Is it convention in tech writing to include the Oxford comma in a two-element list? If so, I'll probably need further reminders :-)